modules/social_features/social_event/templates/event-request-enroll-dialog.html.twig

```diff
@@ -0,0 +1,6 @@
+<div>
+  <div class="col-md">
+    <h5>{% trans %}Request enrollment{% endtrans %}</h5>
+    <p>Tell me whyyyyy</p>
```

Comment:
Hahaha, what is this?

Reply:
This is already gone in the invites branch!